### PR TITLE
iputils: 20211215 -> 20221126

### PR DESCRIPTION
**Pull request by anthonyroussel**
###### Description of changes

See https://github.com/iputils/iputils/releases/tag/20221126

* Added documentation describing removed tools from the last iputils version
* Remove Sun Microsystems License (is no longer "a dependency" with the 3 removed tools)

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by anthonyroussel** [comment]
501+ rebuilds, let's target staging then

File: pkgs/os-specific/linux/iputils/default.nix [hunk 1]
```diff
@@ -65,23 +66,19 @@ in stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
-    description = "A set of small useful utilities for Linux networking";
     inherit (src.meta) homepage;
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/iputils/default.nix:69]: Please set the homepage explicitly

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
Please remind me in a couple of days when no one else merges this until then

File: nixos/doc/manual/release-notes/rl-2305.section.md [hunk 2]
```diff
@@ -10,6 +10,11 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - Cinnamon has been updated to 5.6, see [the pull request](https://github.com/NixOS/nixpkgs/pull/201328#issue-1449910204) for what is changed.
 
+- The iputils package, which is installed by default, no longer provides the
```
Comment (lheckemann) [inline nixos/doc/manual/release-notes/rl-2305.section.md:13]: I think this is more of a thing for "backwards incompatibilities" than "highlights"?

Comment (anthonyroussel) [reply nixos/doc/manual/release-notes/rl-2305.section.md]: Yes indeed, you're right!
I moved that line to the "backwards incompatibilities" section.

Thanks for the review.

**Pull request merged**